assets/i18n/id.json

```diff
@@ -175,6 +175,18 @@
   "I18N_PLAYER_RATE_EXPLORATION": "Belajar sesuatu yang baru? Bagaimana penilaian anda terhadap eksplorasi ini?",
   "I18N_PLAYER_RATINGS_TOOLTIP": "Nilai",
   "I18N_PLAYER_RECOMMEND_EXPLORATIONS": "Disarankan selanjutnya:",
+  "I18N_PLAYER_REPORT_MODAL_BODY_AD": "Menyesatkan konten, spam atau iklan",
```

Comment:
@anggorodewanto If you get a chance, would you mind taking a quick look at these translations? If you don't have time, no worries! Thanks a lot.